### PR TITLE
feat(file-input): new component

### LINKED ISSUE #2207: DS Component — File Upload
The File Upload component is designed to spec and is ready for code development:

Resources

[Design File](https://www.figma.com/design/Eq6RVl1YlbiMBpFS87idc7/File-Upload-Pattern?node-id=1-16&t=kQEZYhkWVPSmIRpp-1)
[Project Brief](https://docs.google.com/document/d/1skcmlauKjowhWnN1FotsJPKWFqRYUc49mz-4ZFobCkY/edit)
Lead Designer: Randy Bascue

### PR DESCRIPTION
**Pull request by LuLaValva**
- Resolves #2207 

## Description

- Create `ebay-file-input` component

## Screenshots

image

### CODE REVIEW

File: src/components/ebay-file-input/component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+class FileUploadInput extends Marko.Component<Input, State> {
+    onCreate() {
+        this.state = { dragging: false };
+    }
+
+    handleFileChange(event: Event) {
+        const target = event.target as HTMLInputElement;
+        this.emit("input", { files: target.files, originalEvent: event });
+        target.value = "";
```
Comment (agliga) [inline src/components/ebay-file-input/component.ts:34]: Remove, I think we dont need it.

**Review by agliga — COMMENTED** [review]
LGTM overall, just one minor comment and I think its ready to merge

**Review by ArtBlue — APPROVED** [review]
LGTM

**Comment by agliga** [comment]
@LuLaValva Once we remove that `value = ""` code, this should be ready to merge

**Review by agliga — APPROVED** [review]

**Pull request merged**